README.md

```diff
@@ -6,12 +6,16 @@
 
 English | [简体中文](README-zh_CN.md)
 
-Unofficial Rust API connector for Bybit's WebSockets APIs.
+An unopinionated, unofficial Rust API connector for Bybit's WebSocket APIs.
 
 ## Disclaimer
 
 This is an **unofficial** Rust API connector for Bybit's APIs and the user assumes all responsibility and risk for the use of this project.
 
+## Project Goals
+
+This library's chief concern is performance. It assumes nothing about your use-case, and so makes no modification of values received from Bybit.
+
```

Comment:
I am doing some researches on [tower](https://github.com/tower-rs/tower) and [serde](https://github.com/serde-rs/serde) recently. I want to figure out if we could provide a `to_owned` (or any other appropriate name following rust naming convention) which convert the WebSocket responses from a referenced version to an owned version.

Though performance is our first goal, a set of convenient public APIs is also took into account.

Here is an outline about what I want to do in v0.2.0:
1. to figure out if we could provide an `to_owned` (or any other appropriate name following rust naming convention) which convert the WebSocket responses from a referenced version to an owned version.
2. remove duplicate code

To achieve above goals, we should introduce more sophisticated design. Maybe tower and serde can be a source of inspirations.

Reply:
> Though performance is our first goal, a set of convenient public APIs is also took into account.

Gotcha!

> Maybe tower and serde can be a source of inspirations.

I'll  look more into tower to see how it can be of use here